components/wpt-results.html

```diff
@@ -22,20 +22,94 @@
     <style>
       :host {
         display: block;
+        font-size: 16px;
       }
       .spec {
         background-color: #eee;
         padding: 1em;
         margin-bottom: 1em;
         border: solid 1px #ccc;
       }
+      section.search {
+        border-bottom: solid 1px #ccc;
+        padding-bottom: 1em;
+        margin-bottom: 1em;
+      }
+      input.query {
+        font-size: 16px;
+        display: block;
+        padding: 0.5em 0;
+        width: 100%;
+      }
+      .browsers.headers {
+        padding: 1em;
+      }
+      .browsers {
+        display: flex;
+        flex-direction: row-reverse;
+      }
+      .browser {
+        display: inline-block;
+        font-size: 0.8em;
+        min-width: 100px;
+      }
+      .browser img {
+        width: 32px;
+        height: 32px;
+      }
     </style>
 
-    <template is="dom-repeat" items="{{specs}}">
+    <section class="search">
+      <input
+        value="{{query::input}}"
```

Comment:
Alright! Seems like it will be helpful for me to open a Polymer reference when reading this code :-)